anitya/app.py

```diff
@@ -38,22 +37,46 @@
 APP.oid = OpenID(APP)
 
 # Set up the logging
-if not APP.debug:
-    APP.logger.addHandler(anitya.mail_logging.get_mail_handler(
-        smtp_server=APP.config.get('SMTP_SERVER', '127.0.0.1'),
-        mail_admin=APP.config.get('MAIL_ADMIN', 'admin@fedoraproject.org')
+logging_config = {
```

Comment:
I don't mind either way, it's just that I've often seen this dict provided as configuration.

Anyway, we can always start like this and reconsider later if needed.